### PR TITLE
feature: update apps-sdk for queue

### PR DESCRIPTION
**Pull request by ShayElkana**
Improved the SDK for queue, added the ability to run it locally and validate via the SDK the payload secret

### CODE REVIEW

File: apps/quickstart-integrations-monday-code/src/queue-service.js [hunk 1]
```diff
@@ -13,14 +13,14 @@ export const produceMessage = async (message) => {
...
 export const readQueueMessage = ({ body, query }) => {
-    const topicMessageSecret = process.env.MNDY_TOPIC_MESSAGES_SECRET
-    const receivedSecret = query.secret;
-    logger.info(`queue message topic Message Secret: "${topicMessageSecret}"`)
+    const envMessageSecret = process.env.MNDY_TOPIC_MESSAGES_SECRET;
+    logger.info(`expected queue secret value: ${envMessageSecret}`)
     logger.info(`queue message received body ${JSON.stringify(body)}`)
     logger.info(`queue message query params ${JSON.stringify(query)}`)
-    if (receivedSecret !== topicMessageSecret)  {
+    if (!queue.validateMessageOrigin(query.secret))  {
```
Comment (yuhgto) [inline apps/quickstart-integrations-monday-code/src/queue-service.js:20]: The function signature doesn't sound intuitive to me.. "Validate Message Origin" makes me think it takes a message as parameter

How about `validateMessageSecret`?

File: apps/quickstart-integrations-monday-code/.env [hunk 2]
```diff
@@ -1,2 +1,3 @@
 MONDAY_SIGNING_SECRET=abcd1234
-PORT=8080
\ No newline at end of file
+PORT=8080
+MNDY_SERVER_ADDRESS=http://localhost:8080
```
Comment (yuhgto) [inline apps/quickstart-integrations-monday-code/.env:3]: What is this used for?

**Review by yuhgto — CHANGES_REQUESTED** [review]

**Review by yuhgto — APPROVED** [review]
Approving this subject to changing the function name `validateMessageSecret`

File: apps/quickstart-integrations-monday-code/.env (hunk 2 above)
Comment (ShayElkana) [reply apps/quickstart-integrations-monday-code/.env:3]: To run & test it locally, we need to know what is the server local address in order to post the queue item payload

**Pull request merged**